devtools/x/src/cargo.rs

```diff
@@ -277,12 +277,14 @@ pub enum CargoCommand<'a> {
         direct_args: &'a [OsString],
         args: &'a [OsString],
         env: &'a [(&'a str, Option<&'a str>)],
+        skip_sccache: &'a bool,
     },
     Build {
         cargo_config: &'a CargoConfig,
         direct_args: &'a [OsString],
         args: &'a [OsString],
         env: &'a [(&'a str, Option<&'a str>)],
+        skip_sccache: &'a bool,
```

Comment:
`bool` is `Copy` so lets not take a reference to them but just have the type be `bool`